domains.txt

```diff
@@ -617,27 +685,34 @@ sjit.co.uk
 skala.co.uk
 skillrecruit.com
 skillsearch.com
+skypath.co.uk
 smart-co.co.uk
 smartedgesolutions.co.uk
 smartsourcing.plc.uk
 snap.hr
+socode.co.uk
 softwarepersonnel.co.uk
 solagroup.com
 soprasteria.com
+soultek.co.uk
```

Comment:
We work on a partnership basis with you to provide simple and effective answers to complex talent problems with a human touch. We don't do commodity recruitment - it's not our bag.